### ISSUE TITLE
[Theme Request] Rias dark theme

### ISSUE THREAD

**Issue by MangoChocobo** [doki-theme/doki-master-theme#52]
## Who/What do you want to be featured?
   Rias from Highschool DxD

## What type of theme?

- [ ] Light Theme
- [x] Dark Theme

## Additional Information (Optional)

  I believe just having a darker theme of the current rias would be amazing, I currently just edit what you currently have to add atleast the sticker to a dark theme but I think just adding in dark theme version of Rias would be amazing.

**Comment by Unthrottled** [comment]
:thinking: 

Hmm, If I where to make another variant of Rias, I'd make that theme a bit more spicy

Something along the lines of this but more SFW (or maybe I just have a spicy/mild variant like Mai)

![spice levels](http://pa1.narvii.com/5585/9acab4c489bd509869d6ad2ddea8421260faaf86_00.gif)

**Comment by MangoChocobo** [comment]
heart this shit 100x over

**Comment by Unthrottled** [comment]
Hey! Just giving a quick update. Rias will have a `Crimson` and a `Onyx` (for a lack of better term) theme. The crimson variant is the existing theme, and the onyx variant is a really dark red scheme (see glorious screenshot below)

So I decided I lacked culture, and decided to fix that.
In the next version of the themes. Rias will have primary and secondary content pack.

The primary content will be the existing and the secondary content will be this:

![Screenshot from 2021-03-26 18-26-07](https://user-images.githubusercontent.com/15972415/112702329-2180e780-8e61-11eb-9a62-cf772605deb1.png)

I've still got to pick out a sticker, but I'm keeping the background. The final product will look just about like this, I'll be tweaking it as I use it overtime.
